script/verification/VaultVerification.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.24;
+
+import {IVault} from "src/interface/IVault.sol";
+import {Provider} from "src/module/Provider.sol";
+import {Withdrawer} from "src/withdraws/Withdrawer.sol";
+import {Vault} from "src/Vault.sol";
+import {MainnetContracts as MC} from "script/Contracts.sol";
+import {Vm} from "lib/forge-std/src/Vm.sol";
+import {WithdrawerConfig} from "script/config/WithdrawerConfig.sol";
+import {IVaultViewer} from "src/interface/IVaultViewer.sol";
+import {MaxVaultViewer} from "src/utils/MaxVaultViewer.sol";
+
+library VaultVerification {
```

Comment:
Has this code been run? 

Let's please commit code that is checked to be working

Please verify the that the verify script works

Reply:
Yes I've ran it. And it ran successfully

Reply:
For BNB testnet, everything runs smoothly except verification for "viewer.isUnderlyingAsset(WBNB)", this can be fixed by adding the underlying asset on the viewer by the UPDATER_ROLE.